css/style.css

```diff
@@ -358,6 +337,27 @@ form .textarea textarea {
   background-position: right;
 }
 
+.Contact-Us {
+  display: flex;
+  flex-direction: column;
+  margin: 0 24px;
+}
+
+form input {
+  height: 48px;
+  width: 283px;
+}
+
+form {
+  margin-left: 12px;
+}
```

Comment:
- [ ] Kindly ensure the elements in your form are aligned correctly, also the styling of this section should match the Figma design given, below is a comparison of the designs.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/177390624-f0f08ddd-65ab-45ab-9c91-79fbaeab01b7.png) | ![image](https://user-images.githubusercontent.com/61408860/177390777-54baf7cf-5830-4aa4-878f-26c987333e2c.png)